File: projekat/src/app/entities/location/location.ts
export class Location {
    country: string;
    countryShort: string
    city: string;
    address: string;
    lat: number;
    lon: number;

    constructor(country: string, countryShort: string, city: string, address: string, lat: number, lon: number){
        this.country = country;
        this.countryShort = countryShort;
        this.city =  city;
        this.address = address;
        this.lat = lat;
        this.lon = lon;
    }
}

File: BookingService/BookingAPI/Model/Location.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace BookingAPI.Model
{
    public class Location
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public string Country { get; set; }
        [Required]
        public string City { get; set; }
        public string Adress { get; set; }
        public float Lat { get; set; }
        public float Lon { get; set; }
    }
}
